infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/impl/InfinispanRemoteDatastoreProvider.java

```diff
@@ -252,7 +257,7 @@ public HotRodSequenceHandler getSequenceHandler() {
 	}
 
 	public <K, V> RemoteCache<K, V> getCache(String cacheName) {
-		RemoteCache<K,V> cache = hotrodClient.getCache( cacheName );
+		RemoteCache<K,V> cache = hotrodClient.getCache( cacheName, TransactionMode.NONE );
```

Comment:
Should this become a property?

Reply:
The param TransactionMode.NONE should be removed at all. I didn't because some tests not pass yet without it.